README.md

```diff
@@ -9,6 +9,14 @@ TSLint
 
 A linter for the TypeScript language.
 
+"next" distribution
+-------------------
+
+The `next` [branch of the TSLint repo](https://github.com/palantir/tslint/tree/next) tracks the latest TypeScript
+compiler and allows you to lint TS code that uses the latest features of the language. Releases from this branch
+are published to npm with the `next` dist-tag, so you can get the latest dev version of TSLint via
+`npm install tslint@next`.
+
```

Comment:
this part should probably also be added to the master branch, but I'll do that in a separate PR